netbox/core/urls.py

```diff
@@ -25,6 +25,11 @@
     path('jobs/<int:pk>/', views.JobView.as_view(), name='job'),
     path('jobs/<int:pk>/delete/', views.JobDeleteView.as_view(), name='job_delete'),
 
+    # Background Tasks
+    path('background-tasks/', views.BackgroundTasksListView.as_view(), name='background_tasks_list'),
+    path('background-tasks/queues/<int:queue_index>/', views.BackgroundTasksQueueListView.as_view(), name='background_tasks_queues'),
+    path('background-tasks/queues/<int:queue_index>/<str:job_id>/', views.BackgroundTasksJobDetailView.as_view(), name='background_tasks_job_detail'),
```

Comment:
We can simplifying the URL tree a bit:

* `background-queues/` - Queue list
* `background-queues/<id>/` - Tasks list for a specific queue
* `background-tasks/<id>/` - Task detail view